cypress.json

```diff
@@ -2,8 +2,9 @@
   "projectId": "Testing Automation",
   "chromeWebSecurity": false,
   "env": {
-    "username": "admin@test.com",
-    "password": "N6.$!tMr-SBNV4N",
-    "baseUrl": "http://localhost:2368/ghost"
+    "version": "v3.42.5",
```

Comment:
Pls leave the default values remember that we are using docker

Reply:
the "version" property is used to name the folder to save screenshots, it doesn't have to do directly with the version used in the execution